graphqlapi/src/main/java/io/stargate/graphql/web/resources/GraphqlCache.java

```diff
@@ -223,6 +224,23 @@ public void onKeyspaceChanged(
     // It is assumed that the data model will only evolve by deploying new GraphQL schema versions.
   }
 
+  public void put(
```

Comment:
nit: maybe we should name this `putDml()` to be symmetric with `getDml()` (my bad, I didn't think of it when I suggested the name before).

Also: could you place it below `getDml()`, since the two are closely related?